components/cart/edit-item-quantity-button.tsx

```diff
@@ -42,7 +42,7 @@ export default function EditItemQuantityButton({
       }}
       disabled={isPending}
       className={clsx(
-        'ease flex min-w-[36px] max-w-[36px] items-center justify-center border px-2 transition-all duration-200 hover:border-gray-800 hover:bg-gray-100 dark:border-gray-700 dark:hover:border-gray-600 dark:hover:bg-gray-900',
```

Comment:
There's a small layout shift when the width of the number changes. Let's style it so that doesn't happen. Try double digit numbers too. 

https://github.com/vercel/commerce/assets/446260/ffe86893-29dd-4c97-9676-6f61fe9b7da5

Reply:
Let's lighten the border like Figma too. 

![CleanShot 2023-07-07 at 15 34 39@2x](https://github.com/vercel/commerce/assets/446260/370d0b3b-c94e-4c8c-b0e6-f8bb49f9b344)